squad/frontend/views.py

```diff
@@ -102,13 +103,26 @@ def build(request, group_slug, project_slug, version):
         'suite',
     ).order_by('-tests_fail', 'suite__slug')
```

Comment:
maybe a good idea do add ordering by environment name/slug as currently envs show up in random order.

Reply:
done